CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Mapbox welcomes participation and contributions from everyone.
 ## Features ✨ and improvements 🏁
 * Introduce a callback to be invoked when the device compass sensors need to be re-calibrated. ([1513](https://github.com/mapbox/mapbox-maps-android/pull/1513))
 * Add support for `LocationComponentSettingsInterface.pulsingMaxRadius` to follow location's accuracy radius. ([1561](https://github.com/mapbox/mapbox-maps-android/pull/1561))
+* Add experimental api to start animations from a background thread. Start animations with the `CameraAnimationsPlugin.post` function. Background thread animations is enabled with a Manifest metadata entry of `com.mapbox.maps.BackgroundAnimatorThread` and corresponding true value. This is needed for Xiaomi devices to process animations when using Android Auto. ([1605](https://github.com/mapbox/mapbox-maps-android/pull/1605)).
```

Comment:
should be moved under `main` block above, 10.8.0-beta.1 was already released

Reply:
Yeah, I don't think we plan on merging this anytime soon. This PR is helping us surface issues and discuss solutions.

I was adding this in case we wanted to merge `android:name="com.mapbox.maps.BackgroundAnimatorThread"`, because with it disabled there should be no change to the existing experience.

With it enabled it will fix a completely broken experience, but it still contains many issues

Reply:
Moved it so resolving this thread